File: 프로그래머스/unrated/181855. 문자열 묶기/문자열 묶기.js
function solution(strArr) {
    const lengthObj = {};
    strArr.forEach(str => {
        lengthObj[str.length] = lengthObj[str.length] || 0;
        lengthObj[str.length]++;
    })
    return Math.max(...Object.values(lengthObj));
}

File: 프로그래머스/unrated/181855. 문자열 묶기/문자열 묶기.java
class Solution {
    public int solution(String[] strArr) {
        int[] strLength = new int[31];
        
        for (String str : strArr) {
            int length = str.length();
            strLength[length]++;
        }
        
        int max = -1;
        for (int length : strLength) {
            if (max < length) {
                max = length;
            }
        }
        
        
        return max;
    }
}